.github/workflows/ghcr_runtime.yml

```diff
@@ -215,6 +215,7 @@ jobs:
         run: make install-python-dependencies
       - name: Run integration tests
         run: |
+          exit 1
```

Comment:
Testing whether this works. Will remove

Reply:
I tested this by force failing a prereq job and the "fail" case ran and blocked my PR